frontend/src/content/news.json

```diff
@@ -1,9 +1,14 @@
+    },
     {
       "date": "2021-07-20",
       "title": "Metabolic Atlas 2.0",
-      "text": "We are excited to announce official release of Metabolic Atlas 2.0 together with the publication of <a href='https: //doi.org/10.1073/pnas.2102344118' target='_blank'><i>Genome-scale metabolic network reconstruction of model animals as a platform for translational research</i></a> 🎉  </br> Visually, the development efforts have alternated between implementing specific features, for example a highly performant and more lightweight 3D viewer, and a dynamic comparison between models, and implementing incremental improvements to the user experience, for example a faster way to search, random suggestions for the Interaction Partners, and a new home page layout. Moreover, a sleuth of bugs, some introduced by the new technology stack, have been fixed.</br> Content-wise, this 2.0 release integrates 5 new GEMs next to the updated Human-GEM and Yeast-GEM, bringing the total to 7 integrated models. With this, new IDs for reactions and metabolites have been created for 6 of these models. These new IDs have been updated on the manually created 2D maps, replacing the previous IDs.</br> More details can be found <a href='https: //github.com/SysBioChalmers/MetabolicAtlas/releases/tag/2.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
+      "text": "We are excited to announce official release of <b>Metabolic Atlas 2.0</b> together with the publication of <a href='https://doi.org/10.1073/pnas.2102344118' target='_blank'><i>Genome-scale metabolic network reconstruction of model animals as a platform for translational research</i></a> 🎉  </br> Visually, the development efforts have alternated between implementing specific features, for example a highly performant and more lightweight 3D viewer, and a dynamic comparison between models, and implementing incremental improvements to the user experience, for example a faster way to search, random suggestions for the Interaction Partners, and a new home page layout. Moreover, a sleuth of bugs, some introduced by the new technology stack, have been fixed.</br> Content-wise, this 2.0 release integrates 5 new GEMs next to the updated Human-GEM and Yeast-GEM, bringing the total to 7 integrated models. With this, new IDs for reactions and metabolites have been created for 6 of these models. These new IDs have been updated on the manually created 2D maps, replacing the previous IDs.</br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
     },
```

Comment:
For links to `zenodo`, would it be possible to link to the specific release? I realized that both version 2.0 and 2.1 link to the same place.

Reply:
A specific `doi` is obtained only after the release on GitHub, so the way to fix this would be to create an issue and update the previous links. The current link points to the latest release that was archived by Zenodo.